librad/src/net/quic.rs

```diff
@@ -37,6 +37,12 @@ use crate::{
     peer::{self, PeerId},
 };
 
+/// The ALPN protocol(s) for the radicle-link protocol stack.
+///
+/// Not currently of significance, but established in order to allow future
+/// major protocol upgrades.
+const ALPN: &[&[u8]] = &[b"rad/1"];
```

Comment:
Still not quite clear what the proper addressation of the protocol is. Is `link` just the name of the software artifact?